src/AndroidDebugLauncher/InstallPaths.cs

```diff
@@ -121,6 +117,14 @@ out matchedPath
                 }
             }
 
+            IEnumerable<INDKFilePath> gdbServerPath = NDKPrebuiltFilePath.GDBServerPaths(targetArchitectureName);
+            result.GDBServerPath = GetNDKFilePath(
+                string.Concat("Android-", targetArchitectureName, "-GDBServerPath"),
+                ndkRoot,
+                gdbServerPath,
+                out matchedPath
+                );
```

Comment:
why do path outputs? What's the difference between result.GDBServerPath and out matchedPath?